### PR TITLE
[1.0.0]: First version

### PR DESCRIPTION
**Pull request by meatball133**

### CODE REVIEW

**Comment by ErikSchierboom** [comment]
https://github.com/exercism/terraform/pull/98/ opened to prepare for deployment

**Comment by ErikSchierboom** [comment]
@meatball133 I see a lot of files being moved related to `.appends` and `.github`. Was there something wrong? The state in this PR was still slightly off, which is why I added https://github.com/exercism/powershell-test-runner/pull/2/commits/56fac0024e8bdb82384bcc6c37d84513e5dd82c0

File: tests/example-success/expected_results.json
Comment (ErikSchierboom) [inline tests/example-success/expected_results.json:1]: The correct status is `pass` (not `success`).  See [the docs](https://exercism.org/docs/building/tooling/test-runners/interface).

File: tests/example-success/expected_results.json [hunk 1]
```diff
@@ -1,4 +1,14 @@
 {
-  "version": 1,
-  "status": "pass"
+  "version": 2,
+  "status": "success",
```
Comment (ErikSchierboom) [inline tests/example-success/expected_results.json]:
```suggestion
  "status": "pass",
```

File: tests/example-success/expected_results.json [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+  "tests": [
+    {
+      "name": "stating something",
+      "status": "success"
```
Comment (ErikSchierboom) [inline tests/example-success/expected_results.json]:
```suggestion
      "status": "pass"
```

File: tests/example-success/expected_results.json [hunk 3]
```diff
@@ -1,4 +1,14 @@
...
+    },
+    {
+      "name": "shouting",
+      "status": "success"
```
Comment (ErikSchierboom) [inline tests/example-success/expected_results.json]:
```suggestion
      "status": "pass"
```

File: tests/example-nested-test/expected_results.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
+{
+  "version": 2,
+  "status": "success",
```
Comment (ErikSchierboom) [inline tests/example-nested-test/expected_results.json]:
```suggestion
  "status": "pass",
```

File: tests/example-nested-test/expected_results.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  "tests": [
+    {
+      "name": "zero steps for one",
+      "status": "success"
```
Comment (ErikSchierboom) [inline tests/example-nested-test/expected_results.json]:
```suggestion
      "status": "pass"
```

File: tests/example-nested-test/expected_results.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    },
+    {
+      "name": "divide if even",
+      "status": "success"
```
Comment (ErikSchierboom) [inline tests/example-nested-test/expected_results.json]:
```suggestion
      "status": "pass"
```

File: tests/example-nested-test/expected_results.json [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    },
+    {
+      "name": "even and odd steps",
+      "status": "success"
```
Comment (ErikSchierboom) [inline tests/example-nested-test/expected_results.json]:
```suggestion
      "status": "pass"
```

File: tests/example-nested-test/expected_results.json [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+    },
+    {
+      "name": "large number of even and odd steps",
+      "status": "success"
```
Comment (ErikSchierboom) [inline tests/example-nested-test/expected_results.json]:
```suggestion
      "status": "pass"
```

File: tests/example-nested-test/expected_results.json [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+    },
+    {
+      "name": "zero is an error",
+      "status": "success"
```
Comment (ErikSchierboom) [inline tests/example-nested-test/expected_results.json]:
```suggestion
      "status": "pass"
```

File: tests/example-nested-test/expected_results.json [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+    },
+    {
+      "name": "negative value is an error",
+      "status": "success"
```
Comment (ErikSchierboom) [inline tests/example-nested-test/expected_results.json]:
```suggestion
      "status": "pass"
```

File: tests/example-partial-fail/expected_results.json [hunk 11]
```diff
@@ -1,5 +1,15 @@
...
-  "message": "TODO: replace with correct output"
+  "tests": [
+    {
+      "name": "stating something",
+      "status": "fail",
+      "message": "Message: Expected strings to be the same, but they were different.\nString lengths are both 9.\nStrings differ at index 6.\nExpected: 'Whatever.'\nBut was:  'Whatevah.'\n           ------^\n\nStack-trace: at hey -HeyBob \"Tom-ay-to, tom-aaaah-to.\" | Should -BeExactly \"Whatever.\", /opt/test-runner/tests/example-partial-fail/MiniBob.test.ps1:8\nat <ScriptBlock>, /opt/test-runner/tests/example-partial-fail/MiniBob.test.ps1:8"
+    },
+    {
+      "name": "shouting",
+      "status": "success"
```
Comment (ErikSchierboom) [inline tests/example-partial-fail/expected_results.json]:
```suggestion
      "status": "pass"
```

**Review by ErikSchierboom — COMMENTED** [review]
The output when running the test runner contains something like this:

```shell
Tests Passed: 2, Failed: 0, Skipped: 0 NotRun: 0
0
1
```

What are the `0` and `1` lines at the end?

File: bin/run.sh
Comment (ErikSchierboom) [inline bin/run.sh:1]: This is a nice and clean implementation!

**Comment by meatball133** [comment]
> @meatball133 I see a lot of files being moved related to .appends and .github. Was there something wrong? 

Jermey gave me this repo and I misplaced some files so the ci didn't work

**Comment by meatball133** [comment]
> What are the 0 and 1 lines at the end?

It was a collection object which counts every time something is added. But I changed it to a normal list so that shouldn't appear again.

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**